reuseport.go

```diff
@@ -0,0 +1,45 @@
+// +build !windows
```

Comment:
Please, add the license disclaimer, because you are copied from fasthttp [reuseport](https://github.com/valyala/fasthttp/blob/master/reuseport/reuseport.go) code.

Thanks!